source/puppet/4.10/lang_data_number.md

```diff
@@ -140,26 +152,23 @@ The full signature for `Float` is:
 
 All of these parameters are optional. They must be listed in order; if you need to specify a later parameter, you must specify values for any prior ones.
 
-Position | Parameter        | Data Type | Default Value | Description
----------| -----------------|-----------|---------------|------------
-1 | Min value | `Float` | negative infinity | The minimum value for the float. This parameter accepts the special value `default`, which will use its default value.
-2 | Max value | `Float` | infinity | The maximum value for the float. This parameter accepts the special value `default`, which will use its default value.
-
+Position | Parameter        | Data type | Default value     | Description
+---------| -----------------|-----------|-------------------|-----------------------------
+1        | Minimum value    | `Float`   | negative infinity | The minimum value for the float. This parameter accepts the special value `default`, which uses its default value.
+2        | Maximum value    | `Float`   | infinity          | The maximum value for the float. This parameter accepts the special value `default`, which uses its default value.
```

Comment:
Note - here it is more correct to use "infinity" and "negative infinity", but there is a MIN and MAX for float values as well as specified in the language specification - it is an IEEE standard for 64 bit floating point values. The details are perhaps not very relevant to a puppet programmer since puppet isn't used for calculations where this matters, but could be good to know that you may run against these limits if you multiply some very large floating point numbers. Perhaps just refer to the IEEE standard?